### PR TITLE
feat(is-standard): include clarification of what is standard

### LINKED ISSUE #28: Clarify that nonstandard sets aren't Standard
...that sounds dumb now that I say it.

There's sometimes confusion about Duel Decks or Commander or whatever, where e.g. a new one just released and you don't see it on WIS so you might think that it's missing by accident, instead of just not Standard. We should remove all doubt and specifically say these types of sets never enter Standard.

### PR DESCRIPTION
**Pull request by forstermatth**
Includes an additional sentence clarifying that non-standard
sets are not in Standard, a tooltip with a specific example
pointing to masters sets, and a link to the official wizards
standard list.

Attempts to fix #28 
From #40

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,8 +113,12 @@
...
       <div class="col-md-7">
         <h2>What <em>is</em> Standard?</h2>
-        <p>Standard is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're
-        released and drop out about eighteen months later.</p>
+        <p>
+          Standard is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're
+          released and drop out about eighteen months later. Some sets are labeled as <strong>Non-standard</strong> or <strong>Modern</strong> and are thus not
+          included in this group. <a class="tip" title="For example, Reprint/Masters sets are Modern sets, not standard sets."></a>
+          The sets that <i>are standard</i> are <a href="http://magic.wizards.com/en/content/standard-formats-magic-gathering">listed here.</a>
+        </p>
```
Comment (glacials) [inline index.html]: I am in favor of the fix, but I'd prefer to shave this down. imo this part

> Reprint/Masters sets are Modern sets, not standard sets.

is the key, and the rest can fall off.

(Also remember that Standard is capitalized)

File: index.html [hunk 2]
```diff
@@ -113,8 +113,11 @@
       </div>
       <div class="col-md-7">
         <h2>What <em>is</em> Standard?</h2>
-        <p>Standard is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're
-        released and drop out about eighteen months later.</p>
+        <p>
+          Standard is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're
+          released and drop out about eighteen months later. Reprint/Masters sets are Modern sets, not standard sets.
```
Comment (glacials) [inline index.html]: Standard is capitalized :P

Comment (forstermatth) [inline index.html]: Ah, my bad. This is fixed.

**Comment by glacials** [comment]
Sweet! Thanks for all the hard work!

**Pull request merged**